python/GangaLHCb/Lib/Applications/BenderBox.py

```diff
@@ -0,0 +1,521 @@
+# -*- coding: utf-8 -*-
```

Comment:
it is highly useful, and e,g it was just vital at SVN epoche. when SVN automatically sustituted some pattersm e.g. authors with not-ASCII symbols (e.g. French names)...  and it caused problem for subsequent python.zip.   Finally I've got a habit to always use this declaration in python code